### PR TITLE
Add com.github.suzie97.communique

### PR DESCRIPTION
**Pull request by Suzie97**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.github.suzie97.communique.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
+pe# This is the same ID that you've used in meson.build and other files
```
Comment (hfiguiere) [inline com.github.suzie97.communique.yml:1]:
```suggestion
# This is the same ID that you've used in meson.build and other files
```

File: com.github.suzie97.communique.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+app-id: com.github.suzie97.communique
+
+# Instead of manually specifying a long list of build and runtime dependencies,
+# we can use a convenient pre-made runtime and SDK. For this example, we'll be
+# using the runtime and SDK provided by elementary.
+runtime: io.elementary.Platform
+runtime-version: '6'
```
Comment (hfiguiere) [inline com.github.suzie97.communique.yml:8]: there is no such runtime on flathub

File: com.github.suzie97.communique.yml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      - type: archive
+        url: https://download.gnome.org/sources/libgee/0.20/libgee-0.20.4.tar.xz
+        sha256: 524c1bf390f9cdda4fbd9a47b269980dc64ab5280f0801b53bc69d782c72de0e
+
+  - name: libpeas
+    buildsystem: meson
+    sources:
+    - type: git
+      url: https://gitlab.gnome.org/GNOME/libpeas.git
+      branch: master
```
Comment (hfiguiere) [inline com.github.suzie97.communique.yml:46]: you'd need a specific tag or commit. not a branch.

File: com.github.suzie97.communique.yml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      url: https://github.com/google/gumbo-parser/archive/v0.10.1.tar.gz
+      sha256: 28463053d44a5dfbc4b77bcf49c8cee119338ffa636cc17fc3378421d714efad
+
+  - name: communique
+    buildsystem: meson
+    config-opts:
+    - '--buildtype=release'
+    sources:
+      - type: dir
+        path: .
```
Comment (hfiguiere) [inline com.github.suzie97.communique.yml:77]: this won't work. there is no source here. This should be either a git repository or an archive.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: com.github.suzie97.communique.yml [hunk 5]
```diff
@@ -0,0 +1,74 @@
+# This is the same ID that you've used in meson.build and other files
+app-id: com.github.suzie97.communique
+
+runtime: org.gnome.Platform
+runtime-version: '40'
```
Comment (hfiguiere) [inline com.github.suzie97.communique.yml:5]:
```suggestion
runtime-version: '41'
```

File: com.github.suzie97.communique.yml [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+# not using hardware like webcams, making sound, or reading external files, we
+# only need permission to draw our app on screen using either X11 or Wayland.
+finish-args:
+  - '--socket=pulseaudio'
+  - '--socket=fallback-x11'
+  - '--share=ipc'
+  - '--socket=wayland'
+  - '--device=dri'
+  - '--share=network'
+  - '--metadata=X-DConf=migrate-path=/com/github/suzie97/communique/'
```
Comment (hfiguiere) [inline com.github.suzie97.communique.yml:22]:
```suggestion
```
This is only needed if the application did exist on an older runtime

File: com.github.suzie97.communique.yml [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+  - '--talk-name=org.gnome.OnlineAccounts'
+  - '--own-name=com.github.suzie97.communique.ArticleView'
```
Comment (hfiguiere) [inline com.github.suzie97.communique.yml:24]:
```suggestion
```

The app already own the name by default.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
bot, build com.github.suzie97.communique

**Comment by hfiguiere** [comment]
graphite is missing.

**Comment by Suzie97** [comment]
bot, build com.github.suzie97.communique

**Comment by Suzie97** [comment]
@hfiguiere Thank you so much for the help!! 😄 😄

File: com.github.suzie97.communique.yml [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+      - -Dexamples=false
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libhandy/1.4/libhandy-1.4.0.tar.xz
+        sha256: 2676d51fa1fa40fdee7497d3e763fefa18b0338bffcd2ee32e7f7e633c885446
+        x-checker-data:
+          type: gnome
+          name: libhandy
+          stable-only: true
+
```
Comment (hfiguiere) [inline com.github.suzie97.communique.yml:114]:
```suggestion
```

This is in the runtime.

File: com.github.suzie97.communique.yml [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+# This section is where you list all the source code required to build your app.
+# If we had external dependencies that weren't included in our SDK, we would list
+# them here.
+modules:
+  - name: libgee
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/libgee/0.20/libgee-0.20.4.tar.xz
+        sha256: 524c1bf390f9cdda4fbd9a47b269980dc64ab5280f0801b53bc69d782c72de0e
```
Comment (hfiguiere) [inline com.github.suzie97.communique.yml:34]: this is also in the runtime

Comment (hfiguiere) [reply com.github.suzie97.communique.yml:34]:
```suggestion
```

**Comment by Suzie97** [comment]
bot, build com.github.suzie97.communique

**Pull request closed without merging**